From e514201a0dc45beb1fd12c210bbef3a26388b5ce Mon Sep 17 00:00:00 2001
From: "Yan, WalterX" <walterx.yan@intel.com>
Date: Thu, 22 Mar 2018 18:25:53 +0800
Subject: [PATCH 15/18] Add permission ACCESS_INSTANT_APPS to Shell.

If the setUp/tearDown of InstantCookieHostTest cannot
clear the instant app data, that will cause the case
testCookieUpdateAndRetrieval fail due to the
cookie's not empty.

Test:
1.run cts -m CtsAppSecurityHostTestCases \
  -t android.appsecurity.cts.InstantCookieHostTest# \
  testCookieUpdateAndRetrieval
2.Check case pass with all abi.

Change-Id: Ia3bb7b61ef146fd3b3926c26c6b72c16ca7b5260
Signed-off-by: Yan, WalterX <walterx.yan@intel.com>
---
 packages/Shell/AndroidManifest.xml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/packages/Shell/AndroidManifest.xml b/packages/Shell/AndroidManifest.xml
index b758e7f5..38d671e 100644
--- a/packages/Shell/AndroidManifest.xml
+++ b/packages/Shell/AndroidManifest.xml
@@ -75,6 +75,7 @@
     <uses-permission android:name="android.permission.CLEAR_APP_CACHE" />
     <uses-permission android:name="android.permission.DELETE_CACHE_FILES" />
     <uses-permission android:name="android.permission.DELETE_PACKAGES" />
+    <uses-permission android:name="android.permission.ACCESS_INSTANT_APPS" />
     <uses-permission android:name="android.permission.ACCESS_SURFACE_FLINGER" />
     <uses-permission android:name="android.permission.READ_FRAME_BUFFER" />
     <uses-permission android:name="android.permission.DEVICE_POWER" />
-- 
1.9.1

